src/OSPSuite.Core/Domain/Services/ParameterIdentifications/IParameterIdentificationTask.cs

```diff
@@ -80,5 +80,11 @@ public interface IParameterIdentificationTask
       /// Returns a clone of the <paramref name="parameterIdentification"/> or null if the user cancels the clone action
       /// </summary>
       ParameterIdentification Clone(ParameterIdentification parameterIdentification);
+
+      /// <summary>
+      /// Updates the <see cref="ParameterIdentification"/>s using the <paramref name="observedData"/>
+      /// Returns an Ienumerable of <see cref="OutputMapping"/> that have been updated
```

Comment:
Update this comment because it's not returning anything now. Also, change to `IReadOnlyList` instead of `IEnumerable`